src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { useState, useCallback } from "react";
+import * as _ from "lodash";
+
+type Props = {
+  button: string,
+  label: string,
+  items: Array<Object>
+};
+
+export default function Dropdown(props: Props) {
+  const { button, label, items } = props;
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [selectedItem, setSelectedItem] = useState(button);
+  const expandMenu = useCallback(() => {
+    setIsExpanded(s => !s);
```

Comment:
Please replace `s` to `isExpanded` or `isShown` - will be more descriptive.

Reply:
Actually - pressing this button should set `isExpanded` always for `true` doesn't it?

Reply:
If click on button another time i want to close menu